modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -1,43 +1,47 @@
-runs in CloudHub.
+You can manage your API from API Manager to apply policies, add contracts, alerts, and SLA tiers. Additionally, you can configure a proxy for your API that runs in CloudHub. You can manage your API using the following steps:
 
-== Creating an API asset in Exchange
+. <<create_an_api, Create an API in Exchange.>>
+. <<import-api-to-API-Manager, Import your API to API Manager.>>
+. <<configure_and_deploy_to_cloudhub, Configure a proxy for your API and deploy to CloudHub.>>
 
-In order to have an API that can be managed through API Manager, the first step consists in creating an API asset in Anypoint Exchange.
+[[create_an_api]]
+== Create an API Asset in Exchange
 
-. Go to Anypoint Exchange https://anypoint.mulesoft.com/exchange
+To manage an API in API Manager, you must first create your API asset in Anypoint Exchange.
+
+To create an API in Exchange:
+
+. Log into Anypoint Platform and select Anypoint Exchange.
 . Click in `New Asset` button
 +
 image::getting-started-proxy-new-asset-button.png[]
 +
-. In `Publish New Asset` view, choose the asset type and an asset name. In case the asset type is RAML, OAS or WSDL, the
-corresponding associated file has to also be specified.
+. In the `Publish New Asset` view, choose the asset type and an asset name. 
++
+For RAML, OAS, or WSDL asset types, you must also provide the corresponding associated file.
 +
 image::getting-started-proxy-new-asset-view.png[width=66%]
 +
-. Click on `Publish button`
+. Click `Publish button`.
 
-Now, we have successfully created an API asset in Exchange.
+[[import-api-to-API-Manager]]
+== Import your API to API Manager
 
-== Managing the API from API Manager
+After you create the API in Exchange, you import the API to API Manager.
 
-Next step consists in importing the created asset to API Manager, so the API can be managed  as in deploy and download proxies,
-apply policies, configure access to it and more.
+To import your API to API Manager:
 
-Follow this steps in order to import your API in API Manager:
-
-. In the API Administration Dashboard click in Manage API > Manage API from Exchange
-+
-image::getting-started-proxy-manage-api-from-exchange.png[manage-api-from-exchange]
+. Log into Anypoint Platform and select API Manager.
 +
-. In the next screen, you have to complete the following fields:
-+
-image::getting-started-proxy-editing-api.png[editing-api,width=75%]
+The API Administration window is displayed.
+. Select *Manage API > Manage API from Exchange*.
+//image::getting-started-proxy-manage-api-from-exchange.png[Manage API from Exchange]
+. On the Manage API from Exchange window, specify the following details:
 +
 - *API Name*: Type the first characters of the API name (The same name you chose when you created your asset in Exchange), and
 then API Manager displays APIs to choose from. Select the API one that was recently created in Exchange.
```

Comment:
"recently created" This seems to imply that any recently API asset created will do, which is not the what we are trying to say. Perhaps: "Select the API Asset that was created in Exchange in the previous steps."

Reply:
I removed that sentence. Done! Thanks!